src/createServer.js

```diff
@@ -1,11 +1,208 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  let users = [];
+  let expenses = [];
+
+  let nextUserId = 1;
+  let nextExpenceId = 1;
```

Comment:
Check what ids you would have. Better use Math.random multiply with new Date.
And move this logic inside of the functions where you use it.